utils/checks.stanza

```diff
@@ -64,14 +64,13 @@ public pcb-check within? (t:Toleranced, value:Toleranced) :
     subcheck-description = "Check that a value is within a given range"
     pass-message =         "Value %_ to %_ is within the range of %_ to %_" % [min-value(t) max-value(t) min-value(value) max-value(value)]
     fail-message =         "Value %_ to %_ is not within the range of %_ to %_" % [min-value(t) max-value(t) min-value(value) max-value(value)]
-    ; locators =             [t value]
+    locators =             []
```

Comment:
Locators can be `JITXObject` or `JITXDef`. Locators do not support for `pcb-structs`.
Let me know if that diverges from MVP.